src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -255,10 +256,21 @@ export default class TableComponent extends ComponentBase implements IComponent,
 			this._table.focus();
 		}
 
+		if (this.checked !== undefined) {
+			this.check(this.checked);
+		}
+
 		this.layoutTable();
 		this.validate();
 	}
 
+	private check(checkInfo: azdata.CheckBoxInfo): void {
+		if (isNullOrUndefined(checkInfo.columnName) || isNullOrUndefined(checkInfo.row)) {
```

Comment:
not sure how I dleted my comment :| ...please add (  || checkInfo.row < 0 || checkInfo.row > this.data.length) also

Reply:
Got this in the commit from the other branch